src/doc/unstable-book/src/language-features/cfg-sanitize.md

```diff
@@ -11,26 +11,24 @@ depending on whether a particular sanitizer is enabled or not.
 
 ## Examples
```

Comment:
I think sanitize should not be in this pull request, it should be another pull request.

Reply:
I don't think this is a significant enough change to warrant it's own PR. However, if `@petrochenkov` or `@varkor` disagree I'll split it into 2 PRs.

Reply:
Yeah, if the change is entirely unrelated to the main theme of the PR, it's better to make a separate PR for it. Pull requests can be made for arbitrarily small changes.

Reply:
Opened #71637